ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/TokenScopeFilterHandler.java

```diff
@@ -35,6 +35,6 @@ public void handle(Context ctx) throws Exception {
 
 	private void sendError(Context ctx) {
 		//Not authorized stop the chain here
-		ctx.getResponse().status(401).send();
+		ctx.getResponse().status(403).send();
```

Comment:
yes but I am not sure that can really ever happen given: https://github.com/SmartThingsOSS/ratpack-bearer-auth/blob/master/ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/BearerTokenAuthHandler.java#L52